.github/workflows/on-pr-and-trunk_one-app-unit-and-lint-tests.yml

```diff
@@ -22,7 +19,7 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version-file: .nvmrc
```

Comment:
lets revert this change and above, apart from that LGTM!!
```suggestion
          node-version: ${{ matrix.node-version }}
```

Reply:
reverted in 527638f 🙇